src/components/CategoriesExpandableList.tsx

```diff
@@ -1,35 +1,51 @@
-import { AccordionPanel, AccordionTitle, AccordionContent, Accordion } from "flowbite-react";
+import {
+  AccordionPanel,
+  AccordionTitle,
+  AccordionContent,
+  Accordion,
+} from "flowbite-react";
 import { categories } from "./categories";
 
-export const CategoriesExpandableList = () => {
-
+const CategoriesExpandableList = () => {
   const renderSubcategories = (categ: any) => {
-    const hasSubcategories = categ.subcategories.length > 0
-    return hasSubcategories ? (<AccordionPanel key={categ.id}>
-    <AccordionTitle>{categ.name}</AccordionTitle>
-    {categ.subcategories.length > 0 ? (<AccordionContent>
-      {categ.subcategories.map((subcateg: any) => (
```

Comment:
No need for checking again the length of the list as you already concluded it has categories in the `hasCategories` condition